shared/metrics/src/test/java/io/pravega/shared/metrics/BasicMetricTest.java

```diff
@@ -16,21 +16,23 @@
 package io.pravega.shared.metrics;
 
 import io.pravega.shared.MetricsTags;
+import io.pravega.test.common.SerializedClassRunner;
```

Comment:
It is odd that it is not recognizing this file as Java, is this just a UI bug?

Reply:
No idea. It's working for me.